Hermes/hermes.py

```diff
@@ -10,6 +10,7 @@
 import tornado.web
 import urllib
 
+import tornado.ioloop
 from tornado.ioloop import IOLoop
```

Comment:
If you import the whole module, you don't need this.
Just remove it and change: IOLoop.instance().start() accordingly.